lib/network/http2_network.go

```diff
@@ -167,14 +63,30 @@ func NewHTTP2Network(config HTTP2NetworkConfig) (h2n *HTTP2Network) {
 	}
 
 	h2n.config = config
-	h2n.handlers = map[string]func(http.ResponseWriter, *http.Request){}
+	h2n.handlers = Handlers{}
 
 	h2n.setNotReadyHandler()
 	h2n.server.ConnState = h2n.ConnState
 
 	return
 }
 
+func (t *HTTP2Network) TLSCertFile() string {
+	return t.tlsCertFile
+}
+
+func (t *HTTP2Network) TLSKeyFile() string {
+	return t.tlsKeyFile
+}
+
+func (t *HTTP2Network) SetTLSCertFile(certFile string) {
+	t.tlsCertFile = certFile
+}
+
+func (t *HTTP2Network) SetTLSKeyFile(keyFile string) {
+	t.tlsKeyFile = keyFile
+}
+
```

Comment:
I am not sure but the funcs(TLSxxx,SetTLSxxx...) seems like not used or called.  If it is right, their funcs can be removed. :-)